vault/logical_system_integ_test.go

```diff
@@ -17,54 +17,58 @@ import (
 	log "github.com/mgutz/logxi/v1"
 )
 
-func TestSystemBackend_enableAuth_plugin(t *testing.T) {
-	coreConfig := &vault.CoreConfig{
-		CredentialBackends: map[string]logical.Factory{
-			"plugin": plugin.Factory,
-		},
-	}
+func TestSystemBackend_Plugin_secret(t *testing.T) {
+	cluster, _ := testSystemBackendMock(t, 1, logical.TypeLogical)
+	defer cluster.Cleanup()
```

Comment:
`cluster.Cleanup()` doesn't clean up backends properly (i.e. doesn't call `backend.Cleanup()`). @briankassouf any suggestions as to how to do this?